packages/arb-bridge-eth/contracts/bridge/SequencerInbox.sol

```diff
@@ -102,9 +100,11 @@ contract SequencerInbox is ISequencerInbox, Cloneable {
                     gasPriceL1,
                     messageDataHash
                 );
+            // Can only force-include after the Sequencer-only window has expired.
             require(l1BlockAndTimestamp[0] + maxDelayBlocks < block.number, "MAX_DELAY_BLOCKS");
             require(l1BlockAndTimestamp[1] + maxDelaySeconds < block.timestamp, "MAX_DELAY_TIME");
 
+            // Ensure that the penultimate message on the delayed queue was the last delayed message added to the sequencer queue
```

Comment:
I think this is more about making sure the message hash calculated above was actually the last message in the chunk of delayed messages we'll be including

This is what we need for the timestamp check since all messages before it in the delayed inbox came at the same time or earlier